README.md

```diff
@@ -74,7 +80,7 @@ make
 make install
 ```
 
-Additionally, standalone drivers for **symPACK** can be built by typing `make examples`
+Additionally, a standalone driver for **symPACK** can be built by typing `make run_sympack2D`.
```

Comment:
Is there any good reason why the `run_sympack2D` driver is not built by the default make target?

Reply:
I guess there’s no reason why it shouldn’t be built by default. I’ll fiddle with this.

Reply:
fixed in [3b0ddd3](https://github.com/symPACK/symPACK/pull/12/commits/3b0ddd3f00759fe368d22285bb33cd3e23a259a9)